src/ng-select/lib/ng-select.component.html

```diff
@@ -28,145 +26,140 @@
 			}
 		}
 
-        @if (multiple && multiLabelTemplate && selectedValues.length > 0) {
-            <ng-template [ngTemplateOutlet]="multiLabelTemplate"
-                         [ngTemplateOutletContext]="{ items: selectedValues, clear: clearItem }">
-            </ng-template>
-        }
+		@if (multiple && multiLabelTemplate && selectedValues.length > 0) {
+			<ng-template [ngTemplateOutlet]="multiLabelTemplate" [ngTemplateOutletContext]="{ items: selectedValues, clear: clearItem }">
+			</ng-template>
+		}
 
-        <div
-            [attr.aria-expanded]="isOpen"
-            [attr.aria-owns]="isOpen ? dropdownId : null"
-            aria-haspopup="listbox"
-            class="ng-input"
-            role="combobox">
-            <input
-                #searchInput
-                (blur)="onInputBlur($event)"
-                (change)="$event.stopPropagation()"
-                (compositionend)="onCompositionEnd(searchInput.value)"
-                (compositionstart)="onCompositionStart()"
-                (focus)="onInputFocus($event)"
-                (input)="filter(searchInput.value)"
-                [attr.aria-activedescendant]="isOpen ? itemsList?.markedItem?.htmlId : null"
-                [attr.aria-controls]="isOpen ? dropdownId : null"
-                [attr.id]="labelForId"
-                [attr.tabindex]="tabIndex"
-                [disabled]="disabled"
-                [readOnly]="!searchable || itemsList.maxItemsSelected"
-                [value]="searchTerm ? searchTerm : ''"
-                aria-autocomplete="list" />
-        </div>
-    </div>
+		<div [attr.aria-expanded]="isOpen" [attr.aria-owns]="isOpen ? dropdownId : null" aria-haspopup="listbox" class="ng-input">
```

Comment:
`aria-haspopup` should normally go in the same element as `role="combobox"`. However by default it already has value of `listbox` if ommited, so I would simply remove it from the code since our popup is of `role="listbox"`. `aria-haspopup` is only required if popup has `role` of a `tree`, `grid` or `dialog`. Based on [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role)